### ISSUE TITLE
Rendering on Moto G(8) Power kind of useless

### ISSUE THREAD

**Issue by atom-smasher** [rignaneseleo/SlimSocial-for-Facebook#227]
Slim Social is working fine on my tablet, but not so fine on my Moto G(8) Power.

Notifications and messages don't show up at all, and most of the rendering makes it unusable.

Moto G(8) Power
Android 10
Resurrection Remix 8.6.9

![slim-be2288d8-f78b-4db7-8016-9766c9fae887](https://github.com/rignaneseleo/SlimSocial-for-Facebook/assets/8199959/7f6a1388-4c65-4be4-9e0e-bf8167bd2884)

![slim-eed1b0e2-4538-46f9-81f1-3be699506d7c](https://github.com/rignaneseleo/SlimSocial-for-Facebook/assets/8199959/803f4608-3702-4b06-a069-b4db758760b4)

![slim-b153d719-f79e-4d1a-9a64-3009e04730e7](https://github.com/rignaneseleo/SlimSocial-for-Facebook/assets/8199959/d5244a9c-9a3f-45f2-8a00-5b0bf3f32f02)

**Comment by atom-smasher** [comment]
Chats: I can see almost everything, except names and messages.

![chats](https://github.com/rignaneseleo/SlimSocial-for-Facebook/assets/8199959/50f1ac99-81c3-42f0-b909-bf54d4f16cee)

**Comment by adsarwal** [comment]
Same issue for me.

**Comment by atom-smasher** [comment]
> Same issue for me.

Same phone? Or about the same size phone?

**Comment by adsarwal** [comment]
Yes, Motorola One 5G Ace.  Perhaps one of their updates broke the app?

**Comment by rignaneseleo** [comment]
Thank you for the report, and I am sorry about the long delay.

This report is against an old version. The app is in active development again since August 2026, and much of it changed: the user agent, a new "Use desktop site" setting, the feed filters, the Messenger screen and photo upload. Old reports are no longer a good reference.

Note: Rendering on a small screen with a 2024 build.

I close this issue to clean up the tracker. Please install **26.09.03+128** or later, from the Releases page, the Play Store or F-Droid. If the problem is still there, open a new issue with your device model, your Android version and the app version.
